app/main.py

```diff
@@ -1,3 +1,60 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: any) -> None:
```

Comment:
The type hint `any` for the `km` parameter should be more specific. Consider using `int` or `float` if those are the expected types.